target/linux/ath79/dts/ar9132_buffalo_wzr-hp-g300nh.dts

```diff
@@ -0,0 +1,257 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
```

Comment:
Sorry, just drive-by comments again:
dts-v1 can be removed now, it's in the ath79.dtsi.